packages/dds/sequence/src/sequence.ts

```diff
@@ -684,7 +683,7 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 								})}`,
 							);
 						}
-						this.processMergeTreeMsg(m);
+						this.processMergeTreeMsg(parseHandles(m, this.serializer));
```

Comment:
I was wondering what about the other call site - but I see that `processCore` takes in an already-decoded object.  Since we don't have expressive types here to distinguish those cases, I'd suggest making it clear in the params comment for each function at this seam whether the message is expected to have handle encoded or decoded.

Reply:
e.g. checkout `loadCatchupOps` in `snapshotLoader.ts`. That's the function that originates the list of messages being iterated over here, from my reading of the code. There's a comment there saying:

> The decoded messages, but handles aren't parsed.  Matches the format that will be passed in SharedObject.processCore

So (A) that's a helpful comment and (B) consider moving this `parseHandles` over there, to maintain the symmetry ("matches the format that will be passed in SharedObject.processCore"). (or if you don't then update that comment to say it doesn't match)